src/App.tsx

```diff
@@ -1,61 +1,97 @@
-import './App.scss';
+import "./App.scss";
+import { useState } from "react";
+import usersFromServer from "./api/users";
+import todosFromServer from "./api/todos";
+import { TodoList } from "./components/TodoList";
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+let isEmptyTitle = false;
+let isEmptySelectedUser = false;
+const users = [...usersFromServer];
+let todos = [...todosFromServer].map((todo) => ({
+  ...todo,
+  user: users.find((user) => user.id === todo.userId),
+}));
+const maxId = todos.map((todo) => todo.id).sort((a, b) => b - a)[0];
 
 export const App = () => {
+  const [newTitle, setNewTitle] = useState("");
+  const [selectedUser, setSelectedUser] = useState(0);
+  const [latstTaskId, setLastTaskId] = useState(maxId);
+  const [afterSubmit, setAfterSubmit] = useState(true);
```

Comment:
Also, the usage of this state is also not clear. The name is confusing and doesn't give any info

Reply:
Changed it slightly